kanvas/src/jsMain/kotlin/HtmlCanvas.kt

```diff
@@ -0,0 +1,226 @@
+package com.juul.krayon.kanvas
+
+import com.juul.krayon.color.Color
+import org.w3c.dom.BEVEL
+import org.w3c.dom.BUTT
+import org.w3c.dom.CENTER
+import org.w3c.dom.CanvasLineCap
+import org.w3c.dom.CanvasLineJoin
+import org.w3c.dom.CanvasRenderingContext2D
+import org.w3c.dom.CanvasTextAlign
+import org.w3c.dom.HTMLCanvasElement
+import org.w3c.dom.LEFT
+import org.w3c.dom.MITER
+import org.w3c.dom.Path2D
+import org.w3c.dom.RIGHT
+import org.w3c.dom.ROUND
+import org.w3c.dom.SQUARE
+import kotlin.math.PI
+
+public class HtmlCanvas(
+    element: HTMLCanvasElement,
+) : Kanvas<Paint, Path2D> {
+
+    private val context = element.getContext("2d") as CanvasRenderingContext2D
+
+    override val width: Float
+        get() = context.canvas.width.toFloat()
+
+    override val height: Float
+        get() = context.canvas.height.toFloat()
+
+    override fun buildPaint(paint: Paint): Paint = paint
+
+    override fun buildPath(actions: PathBuilder<*>.() -> Unit): Path2D =
+        Path2DBuilder().apply(actions).build()
+
+    /** TODO: This can be cleaned up. */
```

Comment:
Can you add a bit more details, like _how_ can it be cleaned up? In case someone else ends up tackling the `TODO`, or way distance future self that doesn't remember _why_ you wrote this comment. 😝